lib/Twig/Test/IntegrationTestCase.php

```diff
@@ -193,7 +193,7 @@ protected function doIntegrationTest($file, $message, $condition, $templates, $e
 
             if (false !== $exception) {
                 list($class) = explode(':', $exception);
-                $this->assertThat(null, new PHPUnit_Framework_Constraint_Exception($class));
+                $this->assertThat(null, new \PHPUnit\Framework\Constraint\Exception($class));
```

Comment:
Thanks! I didn't know the TestCase was available from before! Will do that later today.